lib/web3/eth/personal_module.rb

```diff
@@ -0,0 +1,23 @@
+module Web3
+  module Eth
```

Comment:
My personal style is to match top of the file line breaks with bottom of the file line breaks.

I'd either add some line breaks to the bottom, or remove them from the top.